src/commands.rs

```diff
@@ -662,7 +681,14 @@ pub fn json_set<M: Manager>(manager: M, ctx: &Context, args: Vec<RedisString>) -
                 ))
             }
         }
+    };
+    // if json.set was successful and we have an expire we set it
+    if result.is_ok() {
+        if let Some(ex) = expire {
```

Comment:
We can also support `-1` to allow to `PERSIST` a key which may have a TTL?

Reply:
I'm following Redis semantic, if a user wants  without TTL he should just not send EX or PX